packages/purpose-process/src/resources/templates/documents/riskAnalysisTemplate.html

```diff
@@ -77,9 +77,8 @@ <h1>Analisi del rischio</h1>
     <div class="purpose-answers">{{{answers}}}</div>
 
     <!-- Sigla documento -->
```

Comment:
Remove also the comment 🧹 
(Or, consider adding a different comment for this last section like `<!-- Data documento -->` and if you do that you could add it in all the four templates)

```suggestion
    <!-- Sigla documento -->
```